### DISCUSSION TITLE
Stream is not being removed from the conference room

### DISCUSSION THREAD

**Discussion by jadelacruz** [ant-media/Ant-Media-Server#5286] [FAQs]
Hello, I want to ask a question about the issue I encountered when a stream was disconnected from the internet.  The expected behavior is that the stream should be removed as sub-track of the conference room, but antmedia server still recognizes the stream even though the internet connection of the stream was disconnected.

Basically, `getRoomInfo` still includes the disconnected stream and it is also still displayed as `online` in the dashboard and also it is still included in the `streamsList` when the conference room is being requested via REST API.

**Comment by Mohit-3196** [comment]
Hi @jadelacruz ,
How are you?

Can you please tell me which AMS version are you using?
Is this the issue that you are referring to https://github.com/ant-media/Ant-Media-Server/issues/5143?
If so, then it is merged and available with v2.6.2 https://github.com/ant-media/Ant-Media-Server/releases

If I have misunderstood please let me know.

Thank you

**Reply by jadelacruz** [reply]
Hello @Mohit-3196 
Good day, the AMS version that I'm currently using is `Enterprise Edition 2.5.3 20230107_0851`.
I'll update to the latest version and check if this issue has been resolved.

Thank you for your response.

**Comment by Mohit-3196** [comment]
Sure @jadelacruz ,
Thank you

**Reply by jadelacruz** [reply]
Hello @Mohit-3196 , I tested it on version `Enterprise Edition 2.6.2 20230623_0843` but no luck. To give more idea about what was my test scenario, please check this https://github.com/orgs/ant-media/discussions/5290#discussioncomment-6309977

**Comment by Mohit-3196** [comment]
We are discussing this thread here https://github.com/orgs/ant-media/discussions/5290

FYI all
